perception/occupancy_grid_map_outlier_filter/CMakeLists.txt

```diff
@@ -22,26 +21,26 @@ include_directories(
   ${GRID_MAP_INCLUDE_DIR}
 )
 
-ament_auto_add_library(occupancy_grid_map_outlier_filter SHARED
-  src/occupancy_grid_map_outlier_filter_nodelet.cpp
+ament_auto_add_library(${PROJECT_NAME} SHARED
```

Comment:
@technolojin Shouldn't it be ${PROJECT_NAME} since it is not necessarily one node per package? :pray:

Reply:
1. It is the only node of this package
2. Even the package has multiple nodes, it seems to use shared name https://github.com/autowarefoundation/autoware.universe/blob/b54d57da2a1648e24b04b9249f5f00acb65e763a/perception/image_projection_based_fusion/CMakeLists.txt#L34-L52
3. It follows the Autoware document https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/directory-structure/#package-folder

Is there possibility to add nodes in this package?